keyboards/mokey/mokey12x2/info.json

```diff
@@ -0,0 +1,79 @@
+{
+    "keyboard_name": "Mokey12x2",
+    "manufacturer": "Mokey",
+    "url": "",
+    "maintainer": "qmk",
+    "usb": {
+        "vid": "0x6653",
+        "pid": "0x012A",
+        "device_version": "0.0.1"
+    },
+    "diode_direction": "COL2ROW",
+    "matrix_pins": {
+        "cols": ["A6", "A5", "A4", "A3", "B9"],
+        "rows": ["A2", "A1", "A0", "A7"]
+    },
+    "rgblight": {
+        "pin": "B1",
+        "led_count": 19,
+        "animations": {
+            "alternating": true,
+            "breathing": true,
+            "christmas": true,
+            "knight": true,
+            "rainbow_mood": true,
+            "rainbow_swirl": true,
+            "rgb_test": true,
+            "snake": true,
+            "static_gradient": true,
+            "twinkle": true
+        }
+    },
+    "encoder": {
+        "rotary": [
+            {"pin_a": "B12", "pin_b": "B13"},
+            {"pin_a": "B14", "pin_b": "B15"}
+        ]
+    },
+    "processor": "STM32F103",
+    "bootloader": "stm32duino",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgblight": true,
+        "encoder": true
+    },
+    "layouts": {
+        "LAYOUT": {
+            "layout": [
+                {"matrix": [0, 0], "x":0.25, "y":2.75},
```

Comment:
The keys in this layout are not in the right order. The keymaps may also need to be adjusted.
![image](https://user-images.githubusercontent.com/4781841/235716003-029dcfda-c0a6-4110-94d4-53bf25be37f9.png)